src/kvirc/kernel/KviOptions.cpp

```diff
@@ -667,15 +667,15 @@ KviFontOption g_fontOptionsTable[KVI_NUM_FONT_OPTIONS] = {
 	FONT_OPTION("Notifier", "Lucida Grande", 12, KviOption_sectFlagNotifier | KviOption_resetUpdateNotifier),
 	FONT_OPTION("NotifierTitle", "Lucida Grande", 12, KviOption_sectFlagNotifier | KviOption_resetUpdateNotifier)
 #else
-	FONT_OPTION("IrcView", "Monospace", 10, KviOption_sectFlagIrcView | KviOption_resetUpdateGui),
-	FONT_OPTION("Input", "Sans Serif", 12, KviOption_sectFlagInput | KviOption_resetUpdateGui),
+	FONT_OPTION("IrcView", "Sans Serif", 10, KviOption_sectFlagIrcView | KviOption_resetUpdateGui),
```

Comment:
fixed in https://github.com/kvirc/KVIrc/pull/2029/commits/fd7803b455ffcbb79221dd4899512d96b129a899